manifests/install.pp

```diff
@@ -37,8 +37,7 @@
 
   package { 'mackerel-agent':
     ensure => $ensure
-  }
-
+  } ->
```

Comment:
In order to clarify, please use the `require` instead of `~>` at `/etc/mackerel-agent/conf.d` resource.
See an [example](https://github.com/Tomohiro/puppet-mackerel_agent/blob/250c6a81e4584cd2aa70e7276ee45dc68b8a0e3d/manifests/init.pp#L92)